tests/storage/migration.go

```diff
@@ -1202,37 +1216,27 @@ func createSmallImageForDestinationMigration(vm *virtv1.VirtualMachine, name, si
 }
 
 func waitMigrationToExist(virtClient kubecli.KubevirtClient, vmiName, ns string) {
-	Eventually(func() bool {
+	Eventually(func() []virtv1.VirtualMachineInstanceMigration {
 		ls := labels.Set{
 			virtv1.VolumesUpdateMigration: vmiName,
 		}
-		migList, err := virtClient.VirtualMachineInstanceMigration(ns).List(context.Background(),
-			metav1.ListOptions{
-				LabelSelector: ls.String(),
-			})
+		migList, err := virtClient.VirtualMachineInstanceMigration(ns).List(context.Background(), metav1.ListOptions{
+			LabelSelector: ls.String(),
+		})
 		Expect(err).ToNot(HaveOccurred())
-		if len(migList.Items) < 0 {
-			return false
-		}
-		return true
-
-	}, 120*time.Second, time.Second).Should(BeTrue())
+		return migList.Items
+	}, 120*time.Second, time.Second).Should(BeEmpty())
```

Comment:
If I understand correctly before the function expected to have Items > 0 i.e Should **NOT** be empty